api/src/pcapi/use_cases/save_venue_bank_informations.py

```diff
@@ -126,7 +126,13 @@ def execute(self, application_id: str, procedure_id: Optional[str] = None) -> Op
                 update_demarches_simplifiees_text_annotations(
                     application_details.dossier_id,  # type: ignore [arg-type]
                     application_details.annotation_id,
-                    "Dossier imported Sucessfully",
+                    "Dossier imported sucessfully",
```

Comment:
"su**cc**essfully" (avec 2 "c")

Et tant que j'y suis à pinailler, je mettrais l'adverbe devant : 

```suggestion
                    "Dossier successfully imported",
```